src/aiida_pseudo/cli/family.py

```diff
@@ -85,8 +85,15 @@ def cmd_family_cutoffs_set(family, cutoffs, stringency, unit):  # noqa: D301
     except ValueError as exception:
         raise click.BadParameter(f'`{cutoffs.name}` contains invalid JSON: {exception}', param_hint='CUTOFFS')
 
+    cutoffs_dict = {}
+    for element, values in data.items():
+        try:
+            cutoffs_dict[element] = {'cutoff_wfc': values['cutoff_wfc'], 'cutoff_rho': values['cutoff_rho']}
+        except KeyError as exception:
```

Comment:
I don't think the original exception message will be included this way, will it? So if this exception is raised, it is not clear which key is missing. Of course people could look at the docstring to figure out that `cutoff_wfc` and `cutoff_rho` must be used, but maybe just including the original error might be easier.

Reply:
Indeed, much clearer with the change in your other comment:
```console
$ aiida-pseudo family cutoffs set -s test -u Ry test-family metadata.json 
Usage: aiida-pseudo family cutoffs set [OPTIONS] FAMILY CUTOFFS
Try 'aiida-pseudo family cutoffs set -h' for help.

Error: Invalid value for CUTOFFS: `metadata.json` is missing cutoffs for element `Ag`: 'cutoff_wfc'
```